packages/ERTP/src/amountMath.js

```diff
@@ -173,8 +174,24 @@ const coerceLR = (h, leftAmount, rightAmount) => {
   return [h.doCoerce(leftAmount.value), h.doCoerce(rightAmount.value)];
 };
 
-/** @type {AmountMath} */
+/**
+ * Logic for manipulating amounts.
+ *
+ * Amounts are the canonical description of tradable goods. They are manipulated
+ * by issuers and mints, and represent the goods and currency carried by purses
+ * and
+ * payments. They can be used to represent things like currency, stock, and the
+ * abstract right to participate in a particular exchange.
```

Comment:
Was this text moved from somewhere? Or do I need to review it as novel stuff?

Reply:
I separated the commits to make it more reviewable. I should have given that in the description: [10d31eb](https://github.com/Agoric/agoric-sdk/pull/4551/commits/10d31eb5bfca9f4136453f0d6f7770a6786403cd) isolates the transfer of docs. I neglected to remove the old ones because I wasn't sure yet what else depended on them because my local env is a little borked. I'll rebase to change that to an edit and see how CI goes.